lib/ansible/config/manager.py

```diff
@@ -28,8 +29,9 @@
 from ansible.module_utils.parsing.convert_bool import boolean
 from ansible.parsing.quoting import unquote
 from ansible.utils import py3compat
-from ansible.utils.path import unfrackpath
+from ansible.utils.path import cleanup_tmp_dir
 from ansible.utils.path import makedirs_safe
+from ansible.utils.path import unfrackpath
```

Comment:
This should probably just be:
```
from ansible.utils.path import cleanup_tmp_dir, makedirs_safe, unfrackpath
```

Not sure why it was on 2 lines previously.

Reply:
I just continued that style. I'll flatten it, though. I like that better.